content/about/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: About Proof of Stake
+menu: "main"
+---
+
+Proof of Stake is a consensus mechniasm used in blockchain as an alternative to Proof of Work.
```

Comment:
page looks good @ro-savage... this was the only thing I noticed _mechanism_ instead of 'mechniasm' 🐎

Reply:
Fixed